scrapy/spidermiddlewares/referer.py

```diff
@@ -323,15 +323,17 @@ def _load_policy_class(
     try:
         return cast(Type[ReferrerPolicy], load_object(policy))
     except ValueError:
-        try:
-            return _policy_classes[policy.lower()]
-        except KeyError:
-            msg = f"Could not load referrer policy {policy!r}"
-            if not warning_only:
-                raise RuntimeError(msg)
-            else:
-                warnings.warn(msg, RuntimeWarning)
-                return None
+        tokens = [token.strip() for token in policy.lower().split(",")]
+        for token in tokens[::-1]:
```

Comment:
For future reference, it was not obvious to me why we are picking the last known policy instead of the first one.

```suggestion
        # https://www.w3.org/TR/referrer-policy/#parse-referrer-policy-from-header
        for token in tokens[::-1]:
```